rasa_core/dispatcher.py

```diff
@@ -71,6 +71,23 @@ def utter_message(self, text):
         self.latest_bot_messages.append(bot_message)
         self.output_channel.send_text_message(self.sender_id, text)
 
+    def utter_send_quick_replies(self, text, quick_replies):
+        self.latest_bot_messages.append(BotMessage(text=text,
+                                        data={"quick_replies": quick_replies}
+                                        ))
+        self.output_channel.send_quick_replies(self.sender_id,
+                                               text,
+                                               quick_replies)
+
+    def utter_send_template(self, attachment):
+        self.latest_bot_messages.append(BotMessage(text=None,
+                                        data={"attachment": attachment}
+                                        ))
+        self.output_channel.send_template(self.sender_id, attachment)
```

Comment:
what about the send_list/send_generic_templates?

Reply:
It seems they are not in the dispatcher code. As I've got a deadline on Monday, I have no time to add the proper lines:

def utter_send_list_template(self, attachment):
        self.latest_bot_messages.append(BotMessage(text=None, data={"attachment": attachment}))
        self.output_channel.send_list_template(self.sender_id, attachment)

    def utter_send_generic_template(self, attachment, meta= None, square=False):
        self.latest_bot_messages.append(BotMessage(text=None, data={"attachment": attachment}))
        self.output_channel.send_generic_template(self.sender_id, attachment, meta,square)

Reply:
that's fine, it's not urgent, feel free to make the changes whenever you have time

Reply:
@advl03 any chance of updating this?

Reply:
Updated both dispatcher and changelog. Sorry for the delay!